operator/src/main/resources/Operator.properties

```diff
@@ -182,6 +182,9 @@ WLSDO-0003=No volume mount contains path for log home ''{0}''
 WLSDO-0004=The mount path ''{0}'', in entry ''{1}'' of domain resource ''additionalVolumeMounts'', is not valid
 WLSDO-0005=Environment {0} {1}, specified under ''{2}'', {3} reserved for use by the operator
 WLSDO-0006=Bad namespace for weblogicCredentialsSecret: ''{0}''. If specified, must be same as domain namespace.
+WLSDO-0007=Cluster ''{0}'' specified in the domain resource does not exist in the WebLogic domain
+WLSDO-0008=Managed Server ''{0}'' specified in the domain resource does not exist in the WebLogic domain
```

Comment:
maybe change "WebLogic domain" --> "WebLogic domain home configuration" in both logs to help disambiguate a little

Reply:
Updated to:
WLSDO-0007=Cluster ''{0}'' specified in the domain resource does not exist in the WebLogic domain home configuration.
WLSDO-0008=Managed Server ''{0}'' specified in the domain resource does not exist in the WebLogic domain home configuration.